<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopoChroniaInterpolateRasterDialog</class>
 <widget class="QDialog" name="TopoChroniaInterpolateRasterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TopoChronia Phase II: Interpolate Raster</string>
  </property>
  <widget class="QWidget" name="layoutWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>-10</y>
     <width>993</width>
     <height>612</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>481</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="nodes_age_listWidget"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="Action_CreateNodesAgeList_pushButton">
         <property name="text">
          <string>01 - Create Age List from Nodes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Action_InterpolateRaster_pushButton">
         <property name="text">
          <string>02 - Interpolate Raster</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Action_CorrectWaterLoad_pushButton">
         <property name="text">
          <string>03 - Correct Water Load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Action_FinalRaster_pushButton">
         <property name="text">
          <string>04 - Interpolate Final Raster</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Action_ProcessEverything_pushButton">
         <property name="styleSheet">
          <string notr="true">background-color: #5DAE8B;
color: white;</string>
         </property>
         <property name="text">
          <string>Directly Process All Steps</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="button_box">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>TopoChroniaInterpolateRasterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>TopoChroniaInterpolateRasterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
